src/map.ts

```diff
@@ -161,7 +156,7 @@ export class T7Map {
      * if the parameters in udefined the function toggles the displays
      */
     showLog(show) {
-        if (Form.activeForm) return
+        if (this.shell.activeForm) return
```

Comment:
I'm not sure the line is valid. I know you haven't changed the logic but still, it's quite annoying to press the log button and nothing happens.